### ISSUE TITLE
ShellUTest fails because of NumberNode formatting change

### ISSUE THREAD

**Issue by vsbogd** [opencog/cogserver#22]
I tried to run ShellUTest locally and it looks like it fails because trailing zeroes are absent in `NumberNode` format. At least I don't see them in failure message but see them in example from code of the unit test. And size difference is exactly 7 symbols:

```
/home/vital/projects/opencog/cogserver-build/tests/shell/../../../cogserver/tests/shell/ShellUTest.cxxtest:76: Error: Expected (281 <= reso.size()), found (281 > 274)
Error: Create rep 29 of 30 in thread 16 failed
	Got: 274 >>>opencog> Entering scheme shell; use ^D or a single . on a line by itself to exit.
guile> (EvaluationLink
   (PredicateNode "visible face")
   (ListLink
      (NumberNode "26039")
   )
)

guile> 
guile> <<<
```

**Comment by vsbogd** [comment]
I cannot check the fix because of #5 which reproduces on my machine constantly many requests just doesn't have responses at all.

**Comment by linas** [comment]
Oh, interesting, OK ... looking...

**Comment by linas** [comment]
Yeah, I assume it was opencog/atomspace#2408 that changed the way NumberNodes get printed.  It's harmless without the zeros ... just .. different. It also messed up the attention bank, too.

**Comment by linas** [comment]
For me, this is fixed in #23

**Comment by linas** [comment]
Closing
